pom.xml

```diff
@@ -127,7 +127,7 @@
     <dependency>
       <artifactId>data-model-lib</artifactId>
       <groupId>life.qbic</groupId>
-      <version>2.18.0</version>
+      <version>2.19.0-SNAPSHOT</version>
```

Comment:
I think this will be done by the workflow, isnt it?

Reply:
increase the dependency? I don't think so, needed that Snapshot to test the new model

Reply:
ahh my bad I did not look closely 🙈